tests/terraform/modules/master/instances_server.tf

```diff
@@ -71,10 +71,10 @@ resource "aws_instance" "master2" {
   }
   subnet_id              = var.subnets
   availability_zone      = var.availability_zone
-  vpc_security_group_ids = ["${var.sg_id}"]
+  vpc_security_group_ids = [var.sg_id]
   key_name               = var.ssh_key
-  tags = {
-    Name                              = "${var.resource_name}-servers"
+  tags  =                {
+    Name                 = "${var.resource_name}-server${count.index + 1}"
```

Comment:
I forget if terraform is 0 or 1 indexed. If `count.index` starts at 0, can we do `+2` instead? Since these additional server nodes will always be the 2nd, 3rd, ...nth nodes and never `server1`.

Reply:
@rancher-max

in a case of 4 servers lets say,  it will create like that: 

"aws_instance" "master" = ......-server

"aws_instance" "master2" = .....server1  
"aws_instance" "master2" = .....server2  
"aws_instance" "master2" = .....server3  


is that ok for you ?

Reply:
I'm back and forth in my mind -- I like that or `server1, server2, server3, server4`. Let's leave it because that's a small nit in my own head 😄